### ISSUE TITLE
Fx, dev RP: security exception thrown if cookies disabled.

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/persona#1414]
STR
1) Load up Firefox
2) Turn off cookies
3) Open up firebug
4) Open up the dev RP

```
uncaught exception: [Exception... "Security error" code: "1000" nsresult: "0x805303e8 (NS_ERROR_DOM_SECURITY_ERR)" location: "http://192.168.1.88:10001/ Line: 138"]
(?)()include.js (line 629)
include.js()include.js (line 8)
[Break On This Error]   

;WinChan = (function() {
```

**Comment by shane-tomlinson** [comment]
In Nightly, the exception is:

```
The operation is insecure.
...email: (localStorage.loggedInUser === 'null') ? null : localStorage.loggedInUser
```

**Comment by shane-tomlinson** [comment]
This manifests itself on *.myfavoritebooze.org as well.

**Comment by jrgm** [comment]
verified fixed for firefox on stage train-2012.04.11
